lib/compilers/glimmer/index.js

```diff
@@ -12,13 +12,28 @@ export default class GlimmerTemplatePrecompiler extends Filter {
   }
 
   processString(content, relativePath) {
-    let specifier = getTemplateSpecifier(this.rootName, relativePath);
-    return `export default ${this.precompile(content, { meta: { specifier, '<template-meta>': true } })};`;
+    let specifier = getTemplateSpecifier(this.rootName, removeManagerIdFromPath(relativePath));
+    let managerId = getManagerIdFromPath(relativePath);
+    let meta = { specifier };
```

Comment:
That is correct, from what I've been told. It was only for typing reasons. Not needed in actual precompiled templates though.